frontend/src/scenes/insights/utils.tsx

```diff
@@ -445,40 +444,27 @@ export function isQueryTooLarge(query: Node<Record<string, any>>): boolean {
 export function parseDraftQueryFromLocalStorage(
     query: string
 ): { query: Node<Record<string, any>>; timestamp: number } | null {
-    // First try to uncrush the query if it's a JSONCrush query else fall back to parsing it as a JSON
     try {
-        const uncrushedQuery = JSONCrush.uncrush(query)
-        return JSON.parse(uncrushedQuery)
+        return JSON.parse(query)
     } catch (e) {
-        console.error('Error parsing uncrushed query', e)
-        try {
-            return JSON.parse(query)
-        } catch (e) {
-            console.error('Error parsing query', e)
-            return null
-        }
+        console.error('Error parsing query', e)
```

Comment:
Might be good to use a separate key for the object in localStorage to not litter with errors in the console here. Though probably ok to do so as well